docs/Guides/GitOnZOS.md

```diff
@@ -0,0 +1,59 @@
+# Git on z/OS
```

Comment:
is this something that could also be a proposed patch into git ?

Reply:
Yep, once this is approved, I'll also updated the git README